docs/fundamentals/flow/create-flow.md

```diff
@@ -186,6 +186,103 @@ t.start()  # start Flow in new Thread
 e.set()  # set event and stop (unblock) the Flow
 ```
 
+(restart-flow)=
+## Restart
+
+While developing your Flow, it can be useful to have the Flow be restarted automatically as you keep applying changes to the Flow YAML.
+
+For this you can use the `restart` argument for the Flow so that it watches changes in the configuration YAML code and makes sure that the Flow is restarted with the new configuration.
+
+````{admonition} Caution
+:class: caution
+This feature is thought to let the developer iterate faster while developing but is not intended to be used in production environment.
+````
+
+````{admonition} Note
+:class: note
+This feature requires watchfiles>=0.18 package to be installed.
+````
+
+To see how this would work, let's have a Flow defined in a file `flow.yml` with restart option.
```

Comment:
See prior comments on similar text

Reply:
Apply similar comments above to whole file

Reply:
and here?